pkg/webhook/config/common.go

```diff
@@ -0,0 +1,14 @@
+package config
+
+type WebhookConfig struct {
+	URL                  string                `json:"url"`
+	InsecureSkipVerify   bool                  `json:"insecureSkipVerify"`
+	TLSRootCAFile        string                `json:"tlsRootCAFile"`
+	ClientAuthentication *ClientAuthentication `json:"clientAuthentication"`
```

Comment:
How does the web hook behaves if there is client authentication + insecure skip verify.